config/proven_rules.txt

```diff
@@ -933,4 +933,13 @@ ff6ada24-c7f0-4ae5-a7a6-f20ddb7b591f
 ffa28e60-bdb1-46e0-9f82-05f7a61cc06e
 ffa6861c-4461-4f59-8a41-578c39f3f23e
 ffd622af-d049-449f-af5a-0492fdcc3a58
-fff9d2b7-e11c-4a69-93d3-40ef66189767
\ No newline at end of file
+fff9d2b7-e11c-4a69-93d3-40ef66189767
+48d91a3a-2363-43ba-a456-ca71ac3da5c2
+2650dd1a-eb2a-412d-ac36-83f06c4f2282
+224f140f-3553-4cd1-af78-13d81bf9f7cc
+b793a8e6-28a4-4fb8-816e-17a99e4e7b40
+c1dda054-d638-4c16-afc8-53e007f3fbc5
+d0a61a11-57c9-4afc-b940-3f19b60db08e
+f88e112a-21aa-44bd-9b01-6ee2a2bbbed1
+f97a152e-753c-4975-9375-19087fb66f8c
+fe563ab6-ded4-4916-b49f-a3a8445fe280
```

Comment:
```suggestion
fe563ab6-ded4-4916-b49f-a3a8445fe280
238527ad-3c2c-4e4f-a1f6-92fd63adb864
78cc2dd2-7d20-4d32-93ff-057084c38b93
fa0c05b6-8ad3-468d-8231-c1cbccb64fba
c9a88268-0047-4824-ba6e-4d81ce0b907c
fdf135a2-9241-4f96-a114-bb404948f736
```

Reply:
Here is the link for the rules:
- [238527ad-3c2c-4e4f-a1f6-92fd63adb864](https://github.com/Yamato-Security/hayabusa-rules/blob/bd2fd6bb987c5b487dcd2d548dbf42eba0ab6085/sigma/builtin/category/antivirus/av_exploiting.yml#L2)
- [78cc2dd2-7d20-4d32-93ff-057084c38b93](https://github.com/Yamato-Security/hayabusa-rules/blob/bd2fd6bb987c5b487dcd2d548dbf42eba0ab6085/sigma/builtin/category/antivirus/av_password_dumper.yml#L2)
- [fa0c05b6-8ad3-468d-8231-c1cbccb64fba](https://github.com/Yamato-Security/hayabusa-rules/blob/bd2fd6bb987c5b487dcd2d548dbf42eba0ab6085/sigma/builtin/category/antivirus/av_hacktool.yml#L2)
- [c9a88268-0047-4824-ba6e-4d81ce0b907c](https://github.com/Yamato-Security/hayabusa-rules/blob/bd2fd6bb987c5b487dcd2d548dbf42eba0ab6085/sigma/builtin/category/antivirus/av_relevant_files.yml#L2)
- [fdf135a2-9241-4f96-a114-bb404948f736](https://github.com/Yamato-Security/hayabusa-rules/blob/bd2fd6bb987c5b487dcd2d548dbf42eba0ab6085/sigma/builtin/category/antivirus/av_webshell.yml#L2)